upgrade/scripts/common/ncn-rebuild-common.sh

```diff
@@ -80,122 +80,134 @@ else
+    csi handoff bss-update-param --delete rd.live.overlay.reset --limit "${TARGET_XNAME}"
+    csi handoff bss-update-param --set rd.live.overlay.reset=1 --limit "${TARGET_XNAME}"
+  } >> "${LOG_FILE}" 2>&1
+    record_state "${state_name}" "${target_ncn}"
 else
     echo "====> ${state_name} has been completed"
 fi
-{
-    target_ncn_mgmt_host="${target_ncn}-mgmt"
-    if [[ ${target_ncn} == "ncn-m001" ]]; then
-        target_ncn_mgmt_host=$(ssh -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null ncn-m001 "ipmitool lan print | grep 'IP Address' | grep -v 'Source'"  | awk -F ": " '{print $2}')
-    fi
-    echo "mgmt IP/Host: ${target_ncn_mgmt_host}"
-
-    # retrieve IPMI username/password from vault
-    VAULT_TOKEN=$(kubectl get secrets cray-vault-unseal-keys -n vault -o jsonpath='{.data.vault-root}' | base64 -d)
-    # Make sure we got a vault token
-    [[ -n ${VAULT_TOKEN} ]]
-
-    # During worker upgrades, one vault pod might be offline, so we look for one that works.
-    # List names of all Running vault pods, grep for just the cray-vault-# pods, and try them in
-    # turn until one of them has the IPMI credentials.
-    IPMI_USERNAME=""
-    IPMI_PASSWORD=""
-    for VAULT_POD in $(kubectl get pods -n vault --field-selector status.phase=Running --no-headers \
-                        -o custom-columns=:.metadata.name | grep -E "^cray-vault-(0|[1-9][0-9]*)$") ; do
-        IPMI_USERNAME=$(kubectl exec -it -n vault -c vault ${VAULT_POD} -- sh -c \
-            "export VAULT_ADDR=http://localhost:8200; export VAULT_TOKEN=`echo $VAULT_TOKEN`; \
-            vault kv get -format=json secret/hms-creds/$TARGET_MGMT_XNAME" | 
-            jq -r '.data.Username')
-        # If we are not able to get the username, no need to try and get the password.
-        [[ -n ${IPMI_USERNAME} ]] || continue
-        IPMI_PASSWORD=$(kubectl exec -it -n vault -c vault ${VAULT_POD} -- sh -c \
-            "export VAULT_ADDR=http://localhost:8200; export VAULT_TOKEN=`echo $VAULT_TOKEN`; \
-            vault kv get -format=json secret/hms-creds/$TARGET_MGMT_XNAME" | 
-            jq -r '.data.Password')
-        export IPMI_PASSWORD
-        break
-    done
-    # Make sure we found a pod that worked
-    [[ -n ${IPMI_USERNAME} ]]
-} >> ${LOG_FILE} 2>&1
-state_name="SET_PXE_BOOT"
-state_recorded=$(is_state_recorded "${state_name}" ${target_ncn})
-if [[ $state_recorded == "0" ]]; then
+
+
+state_name="SHUTDOWN_SERVICES"
+state_recorded=$(is_state_recorded "${state_name}" "${target_ncn}")
+if [[ ${state_recorded} == "0" ]]; then
+    rm -f "${basedir}/standdown.sh"
     echo "====> ${state_name} ..."
     {
-        ipmitool -I lanplus -U ${IPMI_USERNAME} -E -H $target_ncn_mgmt_host chassis bootdev pxe options=efiboot
-    } >> ${LOG_FILE} 2>&1
-    record_state "${state_name}" ${target_ncn}
+        if [[ ${target_ncn} == ncn-s* ]]; then
+            cat << 'EOF' > "${basedir}/standdown.sh"
```

Comment:
Probably want a `#!/bin/bash` here... there's no guarantee the default shell hasn't been changed to something other than `bash`.  Same for other heredocs that follow.